students/gotem888/hw-12 searchEngine/css/style.css

```diff
@@ -0,0 +1,276 @@
+@property --rotate {
+    syntax: "<angle>";
+    initial-value: 132deg;
+    inherits: false;
+}
+:root {
+    --form-height: 40vh;
+    --form-width: calc(var(--form-height) / 1.5);
+    --blue: #3498db;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+    margin: 0;
+    min-height: 100vh;
+    background: #191c29;
+  }
+  
+  .wrapper {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-between;
+    align-items: flex-start;
+    gap: 30px;
+    padding: 40px 20px;
+}
+  .form {
+    background: rgba(25, 28, 41, 1);
+    display: flex;
+    flex-flow: row wrap;
+    position: relative;
+    width: 300px;
+    height: var(--card-height);
+    padding: 40px 15px;
+    border-radius: 6px;
+    font-size: 18px;
+    color: rgb(88 199 250 / 100%);
+    cursor: pointer;
+    font-family: Poppins;
+    transition: 2s;
+  }
+  
+  .form:hover {
+    transition: 2s;
+    background-color: #202436;
+    color: rgb(88 199 250 / 100%);
+    box-shadow: 0 0 3px 1px #2e6e80, 0 0 5px 2px #5ddcff;
+  }
+  .form:hover:before, .form:hover:after {
+    animation: none;
+    opacity: 0;
+    transition: 2s;
+  }
+  
+  
+  .form::before {
+    content: "";
+    width: 104%;
+    height: 102%;
+    border-radius: 8px;
+    background-image: linear-gradient(
+      var(--rotate)
+      , #5ddcff, #3c67e3 70%, #4e00c2);
+      position: absolute;
+      z-index: -1;
+      top: -1%;
+      left: -2%;
+      animation: spin 2.5s linear infinite;
+  }
+  
+  .form::after {
+    position: absolute;
+    content: "";
+    top: calc(var(--form-height) / 6);
+    left: 0;
+    right: 0;
+    z-index: -1;
+    height: 100%;
+    width: 100%;
+    margin: 0 auto;
+    transform: scale(0.8);
+    filter: blur(calc(var(--form-height) / 5));
+    background-image: linear-gradient(
+      var(--rotate)
+      , #5ddcff, #3c67e3 43%, #4e00c2, rgb(233, 47, 47));
+      opacity: 1;
+    transition: opacity 2s;
+    animation: spin 2.5s linear infinite;
+  }
+  @keyframes spin {
+    0% {
+      --rotate: 0deg;
```

Comment:
Very unpredictable to see css variables declarations inside animation keyframes. Consider to move it inside `:root{}` selector